src/geometric_algebra/nursery/chisolm.lean

```diff
@@ -96,15 +233,21 @@ exists.elim (vec_sq_scalar (a + b))
       begin
         intros hb ha hab,
         rw [hb, ha, hab] at h1,
-        use kab - ka - kb,
-        repeat {rw ring_hom.map_sub},
-        rw h1,
-        abel,
+        have h2 : (fₛ (kab - ka - kb : G₀) : G) = sym_prod_vec a b, by {
+          repeat {rw ring_hom.map_sub},
+          rw h1,
+          abel,
+        },
+        rw ← h2,
+        rw is_scalar,
+        apply Bᵣ.all_G₀_is_rblade0, -- this feels clumsy, can I make this automatic?
```

Comment:
If you want something to be automatic, just mark the lemma as simp lemma and call simp here, but we need to check if this should fall into that category

Reply:
I think I found that didn't work.